spring-cloud-previews/google-cloud-dialogflow-spring-starter/src/main/java/com/google/cloud/dialogflow/v2/spring/ConversationsSpringAutoConfiguration.java

```diff
@@ -191,6 +191,14 @@ public ConversationsSettings conversationsSettings(
               clientSettingsBuilder.listMessagesSettings().getRetrySettings(), serviceRetry);
       clientSettingsBuilder.listMessagesSettings().setRetrySettings(listMessagesRetrySettings);
 
+      RetrySettings suggestConversationSummaryRetrySettings =
```

Comment:
Corresponding to `conversation.proto` [change](https://github.com/googleapis/googleapis/commit/155e0f4123ba003055587768944a47498c48926b)